packages/perseus/src/widgets/__tests__/radio/base-radio_test.jsx

```diff
@@ -231,32 +268,79 @@ describe("base-radio", () => {
                     generateChoice({
                         content: "Option 1",
                         correct: false,
-                        checked: true,
+                        checked: false,
                     }),
                     generateChoice({
                         content: "Option B",
                         correct: false,
+                        checked: false,
                     }),
                     generateChoice({
                         content: "Option Gamma",
                         correct: true,
+                        checked: true,
                     }),
                     generateChoice({
                         content: "Option Delta",
                         correct: false,
+                        checked: false,
                     }),
                 ],
                 onChange: onChangeHandler,
             });
 
             // Act
-            userEvent.click(screen.getAllByRole("radio")[0]);
+            const radioButton = screen.getByRole("radio", {
+                name: "(Choice C, Checked) Option Gamma",
+            });
+            userEvent.click(radioButton);
 
             // Assert
             expect(updatedValues).toMatchObject({
                 checked: [false, false, false, false],
             });
         });
+
+        it("selects multi-select choice", () => {
+            // Arrange
+            let updatedValues = null;
+            const onChangeHandler = (newValues) => {
+                updatedValues = newValues;
+            };
+
+            renderBaseRadio({
+                multipleSelect: true,
+                choices: [
+                    generateChoice({
+                        content: "Option 1",
+                        correct: false,
+                    }),
+                    generateChoice({
+                        content: "Option B",
+                        correct: false,
+                    }),
+                    generateChoice({
+                        content: "Option Gamma",
+                        correct: true,
+                    }),
+                    generateChoice({
+                        content: "Option Delta",
+                        correct: false,
+                    }),
+                ],
+                onChange: onChangeHandler,
+            });
+
+            // Act
+            userEvent.click(
+                screen.getByRole("checkbox", {name: "(Choice C) Option Gamma"}),
+            );
```

Comment:
So base radio I don't think can handle selecting more than one. Those tests are part of the radio tests. Whenever you click two, the second one is registered, but the first one is not saved.

Reply:
This is likely how the test was implemented, not a limitation of base radio (otherwise how would be ever have multiple choice). It's fine as-is, let's merge the PR.